addons/sys_gsa/CfgWeapons.hpp

```diff
@@ -0,0 +1,47 @@
+#include "script_component.hpp"
+
+class CfgWeapons {
+    class CBA_MiscItem;
+    class CBA_MiscItem_ItemInfo;
+
+    class acre2_vhf30108: CBA_MiscItem {
```

Comment:
Why are those lower-case? We use `ACRE_` for radios and all lower-case, should probably make this the same. (Also easier to move to ACRE3! :P)
```suggestion
    class ACRE_VHF30108: CBA_MiscItem {
```
Same for others classes.